src/animation/component.rs

```diff
@@ -0,0 +1,58 @@
+use bevy::prelude::*;
+
+use std::vec::Vec;
+
+#[allow(dead_code)]
+#[derive(PartialEq, Eq)]
+pub enum AnimationState {
+    Stoped,
+    Playing,
+}
+
+#[derive(Component)]
+pub struct Animator {
+    animation_timer: Timer,
+    animation_state: AnimationState,
+    current_frame: usize,
+    frame_to_sprite_index: Vec<usize>,
```

Comment:
```suggestion
    frame_sprite_indexes: Vec<usize>,
```
Found it more clear (idem for the ones in this file and in the other files)

Reply:
yes !